kani-driver/src/args/mod.rs

```diff
@@ -569,7 +559,7 @@ impl ValidateArgs for VerificationArgs {
         if self.cbmc_args.contains(&OsString::from("--function")) {
             return Err(Error::raw(
                 ErrorKind::ArgumentConflict,
-                "Invalid flag: --function should be provided to Kani directly, not via --cbmc-args.",
+                "Invalid flag: --function is not supported in Kani.",
```

Comment:
Why keep this case? Can't we just treat as any other argument that doesn't exist?

Reply:
This is the CBMC --function args